javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js

```diff
@@ -0,0 +1,48 @@
+//save a note
+const notes = [];
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+//saveNote("pickup", 5);
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+//getNote(5);
+const firstNote = getNote(1);
+//console.log(firstNote); // {content: 'Pick up groceries', id: 1}
+logOutNotesFormatted();
+
+
+
+//saveNote Function
+function saveNote(content, id) {
+  notes.push({
+    content: content,
+    id: id
+  });
+}
+
+
+//Get a note
+function getNote(id) {
+  if (id == '' || typeof(id) == 'string') {
+    console.log('Id neither blank nor string');
+  } else {
+    for (i = 0; i < notes.length; i++) {
+      if (id == notes[i].id) {
+        var idFound = 'found';
```

Comment:
don't use var,
when you have a get function, you need to return the value.

Reply:
ok...will update